polaris-react/src/components/Text/Text.tsx

```diff
@@ -30,11 +30,13 @@ type Variant =
 
 type Alignment = 'start' | 'center' | 'end' | 'justify';
 
-type FontWeight = 'regular' | 'medium' | 'semibold' | 'bold';
+type FontWeight = 'regular' | 'medium' | 'semibold' | 'bold' | null;
 
 type Color = 'success' | 'critical' | 'warning' | 'subdued' | 'text-inverse';
 
-const VariantFontWeightMapping: {[V in Variant]: FontWeight} = {
+const VariantFontWeightMapping: {
```

Comment:
```suggestion
const variantFontWeightMapping: {
```
Light naming suggestion